net-proxy/torsocks/torsocks-2.2.0.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit autotools versionator
+
+MY_PV="$(replace_version_separator 3 -)"
+MY_PF="${PN}-${MY_PV}"
+S=${WORKDIR}/${MY_PF}
+
+DESCRIPTION="Use most socks-friendly applications with Tor"
+HOMEPAGE="https://github.com/dgoulet/torsocks"
+SRC_URI="https://github.com/dgoulet/torsocks/archive/v${MY_PV}.tar.gz -> ${MY_PF}.tar.gz"
+
+LICENSE="GPL-2"
```

Comment:
Per https://github.com/dgoulet/torsocks/blob/master/src/bin/torsocks.in#L13, GPL-2+ should be added to the list as well IMO